src/popup/components/AppsBrowserListItem.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <button class="apps-browser-list-item">
+    <div
+      class="app-image-wrapper"
+    >
+      <IconWrapper
+        v-if="appIcon"
+        :icon="appIcon"
+        class="app-image"
+        is-full
+      />
+      <img
+        v-else-if="appImage"
+        :src="getImageUrl(appImage)"
+        class="app-image"
+      >
+    </div>
+    <div class="app-title">
+      {{ appTitle }}
+    </div>
+  </button>
+</template>
+
+<script lang="ts">
+import { Component, defineComponent, PropType } from 'vue';
+import type { VueI18nTranslation } from 'vue-i18n';
+import IconWrapper from './IconWrapper.vue';
+
+export default defineComponent({
+  name: 'AppsBrowserListItem',
+  components: { IconWrapper },
+  props: {
+    appTitle: { type: String as PropType<string | VueI18nTranslation>, required: true },
```

Comment:
With the new i18n plugin the translation result is treated as a string, so no need to use `VueI18nTranslation` here.